docs/stencil-docs/page-builder/widgets-ui-schema.md

```diff
@@ -0,0 +1,132 @@
+
+## Tabs and sections
+Within the `schema` property of the **Create a Widget Template** endpoint, settings are grouped into `tabs` and `sections`.
+
+### Tabs
+Tabs are a part of the Page Builder schema structure. The schema requires using a single root-level `tab` to contain all visible sections.
+
+>**Note**
+>
+>Sections render in the same order they are listed within a tab.
+
+**Tab schema example**
+```json
+[{
+  "type": "tab",
+  "label": "Content",
+  "sections": [...]
+}]
+```
+
+### Sections
+Sections are groups of related settings. Each section will have a title that the user can collapse. Section labels are optional, but are not collapsible without a label.
+
+Settings render in the same order they are listed within a section.
+
+**Section schema example**
+```json
+[{
+  "type": "tab",
+  "label": "Content",
+  "sections": {
+    "label": "Text",
+    "settings": [...]
+  }
+}]
+```
+
+## Array type
+Arrays allow for building collections of elements within the widget. Array elements live at the top level of the schema. Each element in the array can contain an entire schema internally.
+
+Elements in the list have settings defined by the array’s schema, which can be built using tabs, sections, and settings.
+
+**Array schema example**
+```json
+{
+  "type": "array",
+  "label": "Carousel",
+  "id": "slides",
+  "defaultCount": 2,
+  "entryLabel": "Slide",
+  "thumbnail": "imageUrl",
+  "schema": [
+    {
+      "type": "tab",
+      "label": "Content",
+      "sections": [
+        ...
+        {
+          "type": "imageManager",
+          "id": "imageUrl",
+          "default": {
+            "src": "https://example.com/example.png",
+            "type": "IMAGE_MANAGER"
+          }
+        },
+        ...
+      ]
+    }
+  ]
+}
+
+```
+>**Note**
+>
+>- Each element in the list can have tabs and sections included.
+>- The `defaultCount` attribute is the number of elements in the list to display by default.
+>- The `thumbnail` attribute is used to display an image stored at the specified attribute name.
+>- You can use the `entryLabel` attribute to set a name for each element in the list.
+
+## Hidden settings
+Hidden settings can be used to set up controls which have no user interface drawn in Page Builder. Hidden settings live at the top level of the schema because they are not grouped into any other tabs or arrays. You can use them in advanced widgets that have complex interactions in the preview pane, such as inline editing.
```

Comment:
```suggestion
You can use hidden settings to set up controls which have no user interface drawn in Page Builder. Hidden settings live at the schema's top level because they are not grouped into any other tabs or arrays. You can use them in advanced widgets with complex interactions in the preview pane, such as inline editing.
```